### PR TITLE
Remove lodash from create-pwa

### PR DESCRIPTION
**Pull request by jimbo**
## Description

Remove `lodash` from `@magento/create-pwa`.

## Related Issue

PWA-1452

### Verification Stakeholders

- @dpatil-magento 
- @revanth0212 

### Verification Steps

1. `yarn create` should work

## Screenshots / Screen Captures (if appropriate)

Screen Shot 2021-02-09 at 2 50 53 PM

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.

### CODE REVIEW

**Review by revanth0212 — APPROVED** [review]

**Review by revanth0212 — CHANGES_REQUESTED** [review]
Samplebackends are showing up as `undefined`.

image

**Review by supernova-at — APPROVED** [review]

**Review by jimbo — COMMENTED** [review]
@revanth0212 LGTM

**Review by dpatil-magento — APPROVED** [review]

File: packages/pwa-buildpack/lib/cli/create-project.js [hunk 1]
```diff
@@ -87,8 +87,6 @@ async function findTemplateDir(templateName) {
     }
 }
 
-module.exports.sampleBackends = require('../../sampleBackends.json');
```
Comment (sirugh) [inline packages/pwa-buildpack/lib/cli/create-project.js:90]: Not sure you should delete this just yet - it's still used here: https://github.com/magento/pwa-studio/blob/develop/packages/pwa-buildpack/lib/cli/generate-build-report.js#L12

**Review by sirugh — APPROVED** [review]
this should work - now we just need to test it.

**Review by revanth0212 — APPROVED** [review]
Working when used against `node packages/create-pwa/bin/create-pwa`. Need to publish check against `yarn create @magento/pwa`. Thanks for working on this @jimbo.

**Comment by dpatil-magento** [comment]
Ignore the coveralls build, it triggered because of push event. @jimbo Please check on test failure though.

**Review by sirugh — APPROVED** [review]

**Review by revanth0212 — APPROVED** [review]

**Pull request merged**